pkg/cmd/tests/options.go

```diff
@@ -122,5 +141,17 @@ func (o *TestFrameworkOptions) Complete() error {
 		},
 	}
 
+	if len(o.gcsServiceAccountKeyPath) > 0 {
+		o.objectStorageType = framework.ObjectStorageTypeGCS
+	}
```

Comment:
this is where I'd expect the key file to be read and parsed

Reply:
ok, done